client/webserver/site/src/js/coinexplorers.ts

```diff
@@ -97,20 +112,11 @@ export const CoinExplorers: Record<number, Record<number, (cid: string) => strin
     [Testnet]: (cid: string) => `https://tbch4.loping.net/tx/${cid.split(':')[0]}`,
     [Simnet]: (cid: string) => `https://bch.loping.net/tx/${cid.split(':')[0]}`
   },
-  966: { // matic
-    [Mainnet]: (cid: string) => {
-      const [arg, isAddr] = ethBasedExplorerArg(cid)
-      return isAddr ? `https://polygonscan.com/address/${arg}` : `https://polygonscan.com/tx/${arg}`
-    },
-    [Testnet]: (cid: string) => {
-      const [arg, isAddr] = ethBasedExplorerArg(cid)
-      return isAddr ? `https://amoy.polygonscan.com/address/${arg}` : `https://amoy.polygonscan.com/tx/${arg}`
-    },
-    [Simnet]: (cid: string) => {
-      const [arg, isAddr] = ethBasedExplorerArg(cid)
-      return isAddr ? `https://polygonscan.com/address/${arg}` : `https://polygonscan.com/tx/${arg}`
-    }
-  }
+  966: polygonExplorers,
```

Comment:
Looks like we're missing a `60002: ethExplorers` entry too.

Reply:
Added.